core/src/main/java/com/novoda/noplayer/internal/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -334,4 +336,14 @@ void setMaxVideoBitrate(int maxVideoBitrate) {
         assertIsInPlaybackState();
         NoPlayerLog.w("Tried to set max video bitrate but has not been implemented for MediaPlayer.");
     }
+
+    void attach(AdvertView advertView) {
+        assertIsInPlaybackState();
+        NoPlayerLog.w("Tried to attach advert view but has not been implemented for MediaPlayer.");
```

Comment:
No point doing this for `media player` we will be deprecating it soon enough.

Reply:
Would `assertIsInPlaybackState` crash if we aren't in the right state? What would happen to the log then?